snuba/migrations/snuba_migrations/replays/0001_replays.py

```diff
@@ -65,7 +65,7 @@ def forwards_local(self) -> Sequence[operations.SqlOperation]:
                 columns=raw_columns,
                 engine=table_engines.ReplacingMergeTree(
                     storage_set=StorageSetKey.REPLAYS,
-                    order_by="(project_id, toStartOfDay(timestamp), cityHash64(replay_id), sequence_id)",
+                    order_by="(project_id, toStartOfDay(timestamp), cityHash64(replay_id), timestamp)",
```

Comment:
why do we have both `toStartOfDay(timestamp)` and `timestamp` as part of the sorting key ?

Reply:
ahh i thought sorting keys have to be unique, but they dont. Good point.
>ClickHouse does not require a unique primary key. You can insert multiple rows with the same primary key.
https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/mergetree/

Reply:
(correction: since this a ReplacingMergeTree, the sorting key will be unique)

Reply:
No they definitely have to be unique if you want to preserve those rows. Replacing merge tree merges rows based on this sorting key. But I don't think `timestamp` is guaranteed to be unique - in our other datasets its rounded to the second. Why not segment ID?

Reply:
@lynnagara and I talked off GH -- to handle, we add a new column in addition to segment_id, `event_hash`. for segment updates, this will simply be a hash of the segment_id, and for non_segment updates, it can be a hash of a unique part of the update. (e.g. for event associations, it can be a hash of the event_id). 

This way, we can keep a ReplacingMergeTree (which is useful for deletions (via overwrite), and handling duplicate kafka messages).